packages/logs/lib/models/messages.ts

```diff
@@ -26,19 +27,24 @@ export async function createMessage(row: MessageRow): Promise<void> {
 /**
  * List operations
  */
-export async function listOperations(opts: { limit: number }): Promise<ListOperations> {
+export async function listOperations(opts: { accountId: number; environmentId?: number; limit: number }): Promise<ListOperations> {
+    const q: opensearchtypes.QueryDslQueryContainer = {
```

Comment:
Never a fan of single letter variables. If you call it `query` you can then just do

```
body: { query }
```

later

Reply:
yeah good point 👍🏻